mycroft/client/enclosure/__main__.py

```diff
@@ -51,7 +59,8 @@ def create_enclosure(platform):
     return enclosure
 
 
-def main():
+def main(ready_hook=on_ready, error_hook=on_error):
```

Comment:
Should we include an `stopping_hook` for the enclosure for consistency?
Can't think of why we'd need it currently, but can imagine the need for other hardware enclosures.